app/javascript/src/hyacinth_v1/components/digital_objects/children/ChildStructure.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'react-bootstrap';
+import { useQuery, useMutation } from '@apollo/react-hooks';
+import { useHistory } from 'react-router-dom';
+
+import DigitalObjectInterface from '../DigitalObjectInterface';
+import TabHeading from '../../shared/tabs/TabHeading';
+import { getChildStructureDigitalObjectQuery, updateChildStructureMutation } from '../../../graphql/digitalObjects';
+import ChildStructureEditor from './ChildStructureEditor';
+import GraphQLErrors from '../../shared/GraphQLErrors';
+import FormButtons from '../../shared/forms/FormButtons';
+
+const ChildStructure = (props) => {
+  const { id } = props;
+  const {
+    loading: digitalObjectLoading,
+    error: digitalObjectError,
+    data: digitalObjectData,
+  } = useQuery(getChildStructureDigitalObjectQuery, {
+    variables: { id },
+  });
+  const [updateChildStructure, {
+    error: updateChildStructureErrors,
+  }] = useMutation(
+    updateChildStructureMutation,
+  );
+
+  const history = useHistory();
+  const [childListOrder, setChildListOrder] = useState();
+
+  const onSubmitHandler = () => {
+    const orderedInput = [];
+    if (childListOrder) {
+      childListOrder.forEach((value, i) => {
+        orderedInput.push({ uid: value.id, sortOrder: i });
+      });
+    }
+
+    const parentUid = id;
+    const orderedChildren = orderedInput;
+
+    const variables = {
+      input: {
+        parentUid,
+        orderedChildren,
+      },
+    };
```

Comment:
```suggestion
    const variables = {
      input: {
        parentUid: id,
        orderedChildren: orderedInput,
      },
    };
```
Just a minor suggestion to make this more compact.